Sources/TensorFlow/Operators/NN.swift

```diff
@@ -820,3 +820,51 @@ func _vjpAvgPool3D<Scalar: TensorFlowFloatingPoint>(
     }
   )
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Rearrange depth/space
+//===------------------------------------------------------------------------------------------===//
+
+/// Returns `input` rearranged from depth into blocks of spatial data.
```

Comment:
Unless “rearranged from depth into blocks of spatial data” is a very broadly-understood concept in this domain, I think we need an example, or at least a reference to documentation of a very similar operation.  I (a non-expert in the domain) can't understand what this summary means.

Reply:
I copied examples from `_Raw.depthToSpace`/`_Raw.spaceToDepth`'s documentation.

Reply:
That doesn't make it a good summary 😉

If you know what this thing is actually doing, and can come up with a better description, we'd really appreciate it.  I don't know what it's doing, or I'd suggest something myself.  If you can't think of anything better, I'll file an issue for improving these summaries.

Thanks

Reply:
Updated: https://github.com/tensorflow/swift-apis/pull/710/commits/d8b6203045f19daa71973b98ebf0b101a026c287

From `_Raw.depthToSpace` documentation:
> Rearranges data from depth into blocks of spatial data.
This is the reverse transformation of SpaceToDepth. More specifically,
his op outputs a copy of the input tensor where values from the `depth`
dimension are moved in spatial blocks to the `height` and `width` dimensions.
The attr `block_size` indicates the input block size and how the data is moved.

I took most descriptive part (More specifically ~).
I afraid I can't do better. If it doesn't look good enough, please file an issue.

Reply:
LGreatTM!